code/modules/awaymissions/corpse.dm

```diff
@@ -41,7 +41,7 @@
 	if(!uses)
 		to_chat(user, "<span class='warning'>This spawner is out of charges!</span>")
 		return
-	if(jobban_isbanned(user, banType))
+	if(jobban_isbanned(user, banType) || jobban_isbanned(user, ROLE_SYNDICATE))
```

Comment:
This would also stop people from playing any other ghost roles. I'd rather you make a proc which checks the job bans. Have it default to the ghost role and override it to add the antag role for the spawners which need that

Reply:
As far as I was told by the admin, any ghost roles should be banned by the Syndicate Jobban as the Syndie Jobban greys/reds out the text for ghost roles once applied or some jazz, which indicates that is included in what it bans.

Reply:
I think a headmin should have a decision if it should be applied to all ghostroles, because theres some like the theta station that imo shouldn't be banned